pkgs/applications/virtualization/colima/default.nix

```diff
@@ -1,43 +1,50 @@
 { lib
-, buildGo118Module
+, buildGoModule
 , fetchFromGitHub
 , installShellFiles
 , lima
 , makeWrapper
+, qemu
 }:
 
-buildGo118Module rec {
+buildGoModule rec {
   pname = "colima";
-  version = "0.4.2";
+  version = "0.4.3";
 
   src = fetchFromGitHub {
     owner = "abiosoft";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-66nKH5jxTzLB9bg2lH1E8Cc0GZ6C/N/+yPYhCVEKOBY=";
-
+    sha256 = "tieQOqcbTT6ODUEcLLwWUD7tDOAb6mklbUecjeVjhRc=";
     # We need the git revision
     leaveDotGit = true;
     postFetch = ''
-      git -C $out rev-parse HEAD > $out/.git-revision
+      git -C $out rev-parse --short HEAD > $out/.git-revision
       rm -rf $out/.git
     '';
   };
 
   nativeBuildInputs = [ installShellFiles makeWrapper ];
 
-  vendorSha256 = "sha256-91Ex3RPWxOHyZcR3Bo+bRdDAFw2mEGiC/uNKjdX2kuw=";
+  vendorSha256 = "jDzDwK7qA9lKP8CfkKzfooPDrHuHI4OpiLXmX9vOpOg=";
+
+  CGO_ENABLED = 1;
 
-  doCheck = false;
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/abiosoft/colima/config.appVersion=${version}"
+  ];
 
-  preConfigure = ''
-    ldflags="-X github.com/abiosoft/colima/config.appVersion=${version}
-              -X github.com/abiosoft/colima/config.revision=$(cat .git-revision)"
+  preBuild = ''
+    ldflags+=" -X github.com/abiosoft/colima/config.revision=$(cat .git-revision)"
```

Comment:
I think it's better to define ldflags in one place rather than two like I did it before.

Reply:
@Atemu, works for me.